### DISCUSSION TITLE
PK 7: Time frame?

### DISCUSSION THREAD

**Discussion by guidedways** [mxcl/PromiseKit#1237] [General]
PK 7 with its cancellable promises is just amazing. I see development's been a bit stale on the v7 branch. Do we have a time frame for when it'll move to beta / release?

**Comment by GarthSnyder** [comment]
@mxcl, I'm ready to move ahead with a reconciliation pass whenever you decide the time is right for v7. Still waiting on feedback to #1023.

**Comment by unknown-undefined** [comment]
Any release plan for v7?

**Comment by dougzilla32** [comment]
I'm ready to provide the cancellable extensions when we decide to go ahead with v7.

**Comment by GarthSnyder** [comment]
Please see #1157 for a more general discussion.
